src/Microsoft.AspNetCore.Blazor.Browser.JS/src/Platform/Mono/MonoPlatform.ts

```diff
@@ -26,23 +30,32 @@ export const monoPlatform: Platform = {
   },
 
   findMethod: function findMethod(assemblyName: string, namespace: string, className: string, methodName: string): MethodHandle {
-    // TODO: Cache the assembly_load outputs?
-    const assemblyHandle = assembly_load(assemblyName);
-    if (!assemblyHandle) {
-      throw new Error(`Could not find assembly "${assemblyName}"`);
-    }
-
-    const typeHandle = find_class(assemblyHandle, namespace, className);
-    if (!typeHandle) {
-      throw new Error(`Could not find type "${className}" in namespace "${namespace}" in assembly "${assemblyName}"`);
-    }
-
-    const methodHandle = find_method(typeHandle, methodName, -1);
-    if (!methodHandle) {
-      throw new Error(`Could not find method "${methodName}" on type "${namespace}.${className}"`);
-    }
-
-    return methodHandle;
+        var assemblyHandle: number | undefined = registeredAssembly[assemblyName];
+        if (assemblyHandle) {
```

Comment:
Shouldn't this be `!assemblyHandle`, `!typeHandle` and `!methodHandle`?

Reply:
@campersau yes, your correct this was spotted, but the commit failed as our Internet is currently down :'( and I'm having to use 4G connection at present